lib/Sema/CSApply.cpp

```diff
@@ -6685,26 +6685,9 @@ Expr *ExprRewriter::convertLiteralInPlace(Expr *literal,
                                           DeclName builtinLiteralFuncName,
                                           Diag<> brokenProtocolDiag,
                                           Diag<> brokenBuiltinProtocolDiag) {
-  auto &tc = cs.getTypeChecker();
```

Comment:
Too much deleting here.

Reply:
Yep :)

Reply:
... or not enough deleting further down! See updated version :)